var/spack/repos/builtin/packages/parquet-converters/package.py

```diff
@@ -34,6 +34,7 @@ class ParquetConverters(CMakePackage):
     url      = "ssh://bbpcode.epfl.ch/building/ParquetConverters"
 
     version('develop', git=url, preferred=True)
+    version('0.1', git=url, tag='v0.1')
```

Comment:
wonder what should be policy here: develop preferred or last released as preferred?

Reply:
How will the default modules be built? Do we fix a version? If we just build the default, I would switch it over to the tag.

Reply:
Whatever is mentioned “preferred” will be deployed.

Reply:
I mean, we should deploy the "fixed" version.

* during development or nightly builds, we can explicitly build `@develop` to make sure latest version isn't breaking anything
* with new release, we tag new version and deploy it

Reply:
In that case, I think that `0.1` should be "preferred".

Reply:
OK, fixed